Packs/CommonDashboards/Dashboards/dashboard-79645d69-0da9-45cc-8ed9-4b6f3c6a8095-My_Dashboard_NewandImproved.json

```diff
@@ -0,0 +1,504 @@
+{
+    "fromDate": "0001-01-01T00:00:00Z",
+    "fromDateLicense": "0001-01-01T00:00:00Z",
+    "id": "My Dashboard_NewandImproved",
```

Comment:
we need to change the ID and the name to the real “My Dashboard” - content/Packs/CommonDashboards/Dashboards/dashboard-Home.json

Reply:
is that something someone on your team can modify? Im not sure im the one to do it?